src/sagemaker/jumpstart/utils.py

```diff
@@ -1525,3 +1525,8 @@ def wrapped_f(*args, **kwargs):
     if _func is None:
         return wrapper_cache
     return wrapper_cache(_func)
+
+def get_latest_version(versions: List[str]) -> Optional[str]:
+    try: return None if not versions else max(versions, key=Version)
```

Comment:
When would we return `None`, does this happen in practice!

Reply:
It returns none when the list is empty, this was the existing behavior.